vault-core/src/test/java/org/apache/jackrabbit/vault/packaging/integration/PackageInstallIT.java

```diff
@@ -376,17 +377,17 @@ public void testExternalHookFailsInInstalledPhase() throws RepositoryException,
         assertNodeExists("/testroot");
     }
 
-    /**
-     * Installs a package with no properties
-     */
-    @Test
```

Comment:
I don't think the annotation and the javadoc should be removed

Reply:
That was indeed a merge mistake. Thanks for spotting.